Libraries/Text/RCTShadowText.m

```diff
@@ -119,7 +119,7 @@ - (NSTextStorage *)buildTextStorageForWidth:(CGFloat)width
 
   NSTextContainer *textContainer = [NSTextContainer new];
   textContainer.lineFragmentPadding = 0.0;
-  textContainer.lineBreakMode = _numberOfLines > 0 ? NSLineBreakByTruncatingTail : NSLineBreakByClipping;
```

Comment:
@nicklockwood Perhaps when `this.props.numberOfLines` is `undefined`, it should default to `truncating-tail` instead of `clipping`? It all depends on what `_numberOfLines` defaults to in iOS.